storage/queued_country.hpp

```diff
@@ -6,6 +6,9 @@
 namespace storage
 {
 /// Country queued for downloading.
+/// @TODO(bykoianko) This class assume that a map may consist of one or two mwm files.
+/// But single mwm files are used now. So this class should be redisigned to support
```

Comment:
I do not see why do we need to redesign this class. For backward compatibility, it should always support MapOptions::CarRouting. And even if at some point we will decide to completely delete MapOptions::CarRouting, this class will continue to work fine.